packages/maps/src/components/result/ReactiveOpenStreetMap.js

```diff
@@ -160,7 +160,7 @@ class ReactiveOpenStreetMap extends Component {
 				}}
 				touchZoom
 			>
-				<OpenStreetLayer url="http://{s}.tile.osm.org/{z}/{x}/{y}.png" />
+				<OpenStreetLayer url={this.props.tileServer ? this.props.tileServer : 'http://{s}.tile.osm.org/{z}/{x}/{y}.png'} />
```

Comment:
This can be written as
<OpenStreetLayer url={this.props.tileServer || 'https://{s}.tile.osm.org/{z}/{x}/{y}.png'} />